### PR TITLE
BaseRewardChanged event added

### PR DESCRIPTION
**Pull request by Anze1m**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] index.js exports updated
- [x] Automatic tests added/updated
- [x] Automatic tests pass
- [x] Sanity check passed
- [x] Self-review passed
- [x] Version updated

Note:
Not updated version yet. I guess we will bump version after finishing multisig/multiplexer stuff.

### CODE REVIEW

File: src/wrappers/block_rewards_wrapper.js [hunk 1]
```diff
@@ -14,4 +14,8 @@ export default class BlockRewardsWrapper extends GenesisContractWrapper {
   constructor(blockRewardsContractAddress, web3, defaultAddress) {
     super(blockRewardsContractAddress, contractJsons.blockRewards, web3, defaultAddress);
   }
+
+  async baseRewardChanges(fromBlock, toBlock) {
```
Comment (vanruch) [inline src/wrappers/block_rewards_wrapper.js]: `baseRewardChanged`

**Comment by Anze1m** [comment]
Do not merge this yet, please.
